packages/ui/src/theme/css/component/passwordField.scss

```diff
@@ -1,44 +1,52 @@
-  );
-  --amplify-components-button-disabled-background-color: var(
-    --amplify-components-passwordfield-button-disabled-background-color
-  );
-  --amplify-components-button-disabled-border-color: var(
-    --amplify-components-passwordfield-button-disabled-border-color
-  );
-  --amplify-components-button-disabled-color: var(
-    --amplify-components-passwordfield-button-disabled-color
-  );
-  --amplify-components-button-focus-background-color: var(
-    --amplify-components-passwordfield-button-focus-background-color
-  );
-  --amplify-components-button-focus-border-color: var(
-    --amplify-components-passwordfield-button-focus-border-color
-  );
-  --amplify-components-button-focus-color: var(
-    --amplify-components-passwordfield-button-focus-color
-  );
-  --amplify-components-button-hover-background-color: var(
-    --amplify-components-passwordfield-button-hover-background-color
-  );
-  --amplify-components-button-hover-border-color: var(
-    --amplify-components-passwordfield-button-hover-border-color
-  );
-  --amplify-components-button-hover-color: var(
-    --amplify-components-passwordfield-button-hover-color
-  );
-}
+.amplify-passwordfield {
+  --amplify-components-fieldcontrol-color: var(
+    --amplify-components-passwordfield-color
+  );
+  --amplify-components-button-color: var(
+    --amplify-components-passwordfield-button-color
+  );
+  --amplify-components-button-active-background-color: var(
+    --amplify-components-passwordfield-button-active-background-color
+  );
+  --amplify-components-button-active-border-color: var(
+    --amplify-components-passwordfield-button-active-border-color
+  );
+  --amplify-components-button-active-color: var(
+    --amplify-components-passwordfield-button-active-color
+  );
+  --amplify-components-button-disabled-background-color: var(
+    --amplify-components-passwordfield-button-disabled-background-color
+  );
+  --amplify-components-button-disabled-border-color: var(
+    --amplify-components-passwordfield-button-disabled-border-color
+  );
+  --amplify-components-button-disabled-color: var(
+    --amplify-components-passwordfield-button-disabled-color
+  );
+  --amplify-components-button-focus-background-color: var(
+    --amplify-components-passwordfield-button-focus-background-color
+  );
+  --amplify-components-button-focus-border-color: var(
+    --amplify-components-passwordfield-button-focus-border-color
+  );
+  --amplify-components-button-focus-color: var(
+    --amplify-components-passwordfield-button-focus-color
+  );
+  --amplify-components-button-hover-background-color: var(
+    --amplify-components-passwordfield-button-hover-background-color
+  );
+  --amplify-components-button-hover-border-color: var(
+    --amplify-components-passwordfield-button-hover-border-color
+  );
+  --amplify-components-button-hover-color: var(
+    --amplify-components-passwordfield-button-hover-color
+  );
+}
+
+amplify-field__show-password {

```

Comment:
Could you follow the same naming convention? We are using BEM. So this should be `.amplify-passwordfield__show-password`

Reply:
```suggestion
.amplify-passwordfield {
  &__show-password--error {
      color: var(--amplify-components-fieldcontrol-error-border-color);
      border-color: var(--amplify-components-fieldcontrol-error-border-color);
  }
}
```